### ISSUE TITLE
Fresh installations should not contain "Calendar tables are missing" and "Contacts tables are missing" in log

### ISSUE THREAD

**Issue by LukasReschke** [owncloud/core#23747]
### Steps to reproduce
1. Install master or 9.0
2. Check the ownCloud log

### Expected behaviour
Should be empty or at least have no "Errors" and only "Info" entries.

### Actual behaviour
Contains the following entries with the log level "Error":

> Exception: {"Exception":"DomainException","Message":"Calendar tables are missing. Nothing to do.","Code":0,"Trace":"#0 \/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/lib\/migration\/migratecalendars.php(84): OCA\\Dav\\Migration\\CalendarAdapter->setup()\n#1 \/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/appinfo\/application.php(202): OCA\\Dav\\Migration\\MigrateCalendars->setup()\n#2 \/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/appinfo\/install.php(27): OCA\\Dav\\AppInfo\\Application->migrateCalendars()\n#3 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/installer.php(634): include('\/Users\/lukasres...')\n#4 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/installer.php(585): OC_Installer::includeAppScript('\/Users\/lukasres...')\n#5 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/installer.php(556): OC_Installer::installShippedApp('dav')\n#6 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/Setup.php(370): OC_Installer::installShippedApps()\n#7 \/Users\/lukasreschke\/Documents\/Programming\/oc\/core\/controller\/setupcontroller.php(64): OC\\Setup->install(Array)\n#8 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/base.php(822): OC\\Core\\Controller\\SetupController->run(Array)\n#9 \/Users\/lukasreschke\/Documents\/Programming\/oc\/index.php(39): OC::handleRequest()\n#10 {main}","File":"\/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/lib\/migration\/calendaradapter.php","Line":68}	

> Exception: {"Exception":"DomainException","Message":"Contacts tables are missing. Nothing to do.","Code":0,"Trace":"#0 \/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/lib\/migration\/migrateaddressbooks.php(83): OCA\\Dav\\Migration\\AddressBookAdapter->setup()\n#1 \/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/appinfo\/application.php(186): OCA\\Dav\\Migration\\MigrateAddressbooks->setup()\n#2 \/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/appinfo\/install.php(26): OCA\\Dav\\AppInfo\\Application->migrateAddressbooks()\n#3 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/installer.php(634): include('\/Users\/lukasres...')\n#4 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/installer.php(585): OC_Installer::includeAppScript('\/Users\/lukasres...')\n#5 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/installer.php(556): OC_Installer::installShippedApp('dav')\n#6 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/private\/Setup.php(370): OC_Installer::installShippedApps()\n#7 \/Users\/lukasreschke\/Documents\/Programming\/oc\/core\/controller\/setupcontroller.php(64): OC\\Setup->install(Array)\n#8 \/Users\/lukasreschke\/Documents\/Programming\/oc\/lib\/base.php(822): OC\\Core\\Controller\\SetupController->run(Array)\n#9 \/Users\/lukasreschke\/Documents\/Programming\/oc\/index.php(39): OC::handleRequest()\n#10 {main}","File":"\/Users\/lukasreschke\/Documents\/Programming\/oc\/apps\/dav\/lib\/migration\/addressbookadapter.php","Line":72}
	
<hr/>
cc @DeepDiver1975

**Comment by sekmenhuseyin** [comment]
i have the same problem. i have installed owncloud more than three times, everytime i gpt the same errors.


```
Error	no app in context	Exception: {"Exception":"DomainException","Message":"Calendar tables are missing. Nothing to do.","Code":0,"Trace":"#0 \/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/lib\/migration\/migratecalendars.php(84): OCA\\Dav\\Migration\\CalendarAdapter->setup()\n#1 \/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/appinfo\/application.php(202): OCA\\Dav\\Migration\\MigrateCalendars->setup()\n#2 \/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/appinfo\/install.php(27): OCA\\Dav\\AppInfo\\Application->migrateCalendars()\n#3 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/installer.php(621): include('\/home\/sekmetek\/...')\n#4 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/installer.php(572): OC_Installer::includeAppScript('\/home\/sekmetek\/...')\n#5 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/installer.php(546): OC_Installer::installShippedApp('dav')\n#6 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/setup.php(370): OC_Installer::installShippedApps()\n#7 \/home\/sekmetek\/cloud.sekmetech.net\/core\/controller\/setupcontroller.php(64): OC\\Setup->install(Array)\n#8 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/base.php(823): OC\\Core\\Controller\\SetupController->run(Array)\n#9 \/home\/sekmetek\/cloud.sekmetech.net\/index.php(39): OC::handleRequest()\n#10 {main}","File":"\/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/lib\/migration\/calendaradapter.php","Line":68}	2016-04-04T06:44:28+00:00
Error	no app in context	Exception: {"Exception":"DomainException","Message":"Contacts tables are missing. Nothing to do.","Code":0,"Trace":"#0 \/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/lib\/migration\/migrateaddressbooks.php(83): OCA\\Dav\\Migration\\AddressBookAdapter->setup()\n#1 \/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/appinfo\/application.php(186): OCA\\Dav\\Migration\\MigrateAddressbooks->setup()\n#2 \/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/appinfo\/install.php(26): OCA\\Dav\\AppInfo\\Application->migrateAddressbooks()\n#3 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/installer.php(621): include('\/home\/sekmetek\/...')\n#4 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/installer.php(572): OC_Installer::includeAppScript('\/home\/sekmetek\/...')\n#5 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/installer.php(546): OC_Installer::installShippedApp('dav')\n#6 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/private\/setup.php(370): OC_Installer::installShippedApps()\n#7 \/home\/sekmetek\/cloud.sekmetech.net\/core\/controller\/setupcontroller.php(64): OC\\Setup->install(Array)\n#8 \/home\/sekmetek\/cloud.sekmetech.net\/lib\/base.php(823): OC\\Core\\Controller\\SetupController->run(Array)\n#9 \/home\/sekmetek\/cloud.sekmetech.net\/index.php(39): OC::handleRequest()\n#10 {main}","File":"\/home\/sekmetek\/cloud.sekmetech.net\/apps\/dav\/lib\/migration\/addressbookadapter.php","Line":72}	2016-04-04T06:44:28+00:00
Error	PHP	Undefined offset: 0 at /home/sekmetek/cloud.sekmetech.net/lib/private/setup/mysql.php#46	2016-04-04T06:44:26+00:00
```

**Comment by enoch85** [comment]
ref https://github.com/owncloud/core/issues/23637

**Comment by MohiPE** [comment]
I'm using 9.0.2, and same problem :(

**Comment by DeepDiver1975** [comment]
This is fixed in 9.1 - the migration has been dropped.
For 9.0.4 we have a PR open #25279
